### ISSUE TITLE
Nested tabs issue

### ISSUE THREAD

**Issue by Sonith** [angular-ui/bootstrap#6552]
### Bug description:

First off, great job on everything that was accomplished here. I have saved a lot of development time. 
Second, I apologize if I am using this incorrectly.

When working with nested tabs, I am unable to select a tab.

### Link to minimally-working plunker that reproduces the issue:

https://plnkr.co/edit/gJuQ7WaRZayymO7DH1no?p=preview

### Steps to reproduce the issue:

Click on tab t2

### Version of Angular, UIBS, and Bootstrap

Angular: 1.6.4

UIBS: 2.5.0 - 2017-01-28

Bootstrap:

**Comment by WVWillHall** [comment]
this works:
https://plnkr.co/edit/kGSrPZmiIbTtvBfC7kbr?p=preview

**Comment by Sonith** [comment]
Yeah, I noticed that too. But a lot of developers use the controllerAs syntax. Right?
Or does this framework not support this syntax?

**Comment by WVWillHall** [comment]
whats really gonna trip ya out is this:

https://plnkr.co/edit/gAatLI5g5iXDmCxpexZL?p=preview

just modified the 'active' attributes

**Comment by Sonith** [comment]
Yeah. But do you see why what I reported is an issue?
I try to follow the below style guide.
https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#style-y030
It should work, correct?

**Comment by WVWillHall** [comment]
Yes, it is a bug - you were correct to report it.

**Comment by InFerYes** [comment]
I have a similar issue with controllerAs. The attribute `active="controller.active"` gets set to `null`. When I remove the `active="controller.active"` attribute from the tabset then the value in `controller.active` is no longer `null`. Something is clearing the value but I'm not sure what or where it's happening.
